backend/store/error_groups.go

```diff
@@ -3,13 +3,145 @@ package store
 import (
 	"context"
 	"errors"
+	"strconv"
 	"time"
 
 	"github.com/highlight-run/highlight/backend/model"
 	"github.com/highlight-run/highlight/backend/opensearch"
 	privateModel "github.com/highlight-run/highlight/backend/private-graph/graph/model"
+	"github.com/samber/lo"
 )
 
+type ListErrorObjectsParams struct {
+	After  *string
+	Before *string
+}
+
+// Number of results per page
+const LIMIT = 10
+
+func (store *Store) ListErrorObjects(errorGroup model.ErrorGroup, params ListErrorObjectsParams) (privateModel.ErrorObjectConnection, error) {
+
+	var errorObjects []model.ErrorObject
+
+	query := store.db.
+		Where(&model.ErrorObject{ErrorGroupID: errorGroup.ID}).Limit(LIMIT + 1)
+
+	var (
+		endCursor       string
+		startCursor     string
+		hasNextPage     bool
+		hasPreviousPage bool
+	)
+
+	if params.After != nil {
+		query = query.Order("id ASC").Where("id > ?", *params.After)
+	} else if params.Before != nil {
+		query = query.Order("id DESC").Where("id < ?", *params.Before)
+	} else {
+		query = query.Order("id ASC")
+	}
+
+	if err := query.Find(&errorObjects).Error; err != nil {
+		return privateModel.ErrorObjectConnection{}, err
+	}
+
+	if len(errorObjects) == 0 {
+		return privateModel.ErrorObjectConnection{}, nil
+	}
+
+	// Extract the non-null session IDs
```

Comment:
Because some of the data is on the session (`session.user_properties` and `session.app_version`) (not the error_object), we load the session for each error_object.
Because the session_id is nullable, we have to do some manual work to support preloading.

Reply:
To clarify why we need user_properties, the email in the listview is derived via this logic:

https://github.com/highlight/highlight/blob/1ef0610bc42906347fccb240100939455f3a9dbf/frontend/src/pages/ErrorsV2/ErrorInstance/ErrorInstance.tsx#L504

https://github.com/highlight/highlight/blob/1ef0610bc42906347fccb240100939455f3a9dbf/frontend/src/pages/Sessions/SessionsFeedV3/MinimalSessionCard/utils/utils.ts#L39-L55

meaning it won't always be email -- it's entirely dependent on how much we've identified on the session.